examples/images_contours_and_fields/specgram_demo.py

```diff
@@ -25,16 +24,15 @@
 nse = 0.01 * np.random.random(size=len(t))
 
 x = s1 + s2 + nse  # the signal
-NFFT = 1024       # the length of the windowing segments
+NFFT = 1024  # the length of the windowing segments
 Fs = int(1.0 / dt)  # the sampling frequency
 
-# Pxx is the segments x freqs array of instantaneous power, freqs is
-# the frequency vector, bins are the centers of the time bins in which
-# the power is computed, and im is the matplotlib.image.AxesImage
-# instance
-
-ax1 = plt.subplot(211)
-plt.plot(t, x)
-plt.subplot(212, sharex=ax1)
-Pxx, freqs, bins, im = plt.specgram(x, NFFT=NFFT, Fs=Fs, noverlap=900)
+fig, (ax1, ax2) = plt.subplots(nrows=2)
+ax1.plot(t, x)
+Pxx, freqs, bins, im = ax2.specgram(x, NFFT=NFFT, Fs=Fs, noverlap=900)
+# The `specgram` method returns 4 objects. They are:
```

Comment:
Unfortunately sphinx references don't work in code comments. See the [resulting file](https://8320-1385122-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/gallery/images_contours_and_fields/specgram_demo.html#sphx-glr-gallery-images-contours-and-fields-specgram-demo-py).

I'll leave it up to you, if you want to keep it anyway.